### DISCUSSION TITLE
Remove workflow tests for old Python versions

### DISCUSSION THREAD

**Discussion by wigging** [watertap-org/watertap#811] [General]
Tests in the GitHub Actions workflow are run for Python versions 3.7, 3.8, 3.9, and 3.10. The current version of Python is 3.10 and version 3.11 will be released soon. Can the older Python versions 3.7, 3.8, 3.9 be removed from the tests workflow? This would speed up the workflow for pull requests.

**Comment by ksbeattie** [comment]
Our goal is to support the same versions of Python that our upstream requirements do.  The most significant of those being IDAES (3.7-3.10) and Pyomo (perhaps back to 2.7).
